File: app/src/main/java/com/sdxxtop/zjlguardian/widget/people_picker/data/DepartmentData.java
package com.sdxxtop.zjlguardian.widget.people_picker.data;

import androidx.annotation.Keep;

import java.util.List;

/**
 * Email: zhousaito@163.com
 * Created by zhousaito 2019-09-24 09:22
 * Version: 1.0
 * Description:
 */
@Keep
public class DepartmentData implements IPickerData {
    private int departmentId;
    private String label;
    private String value;
    private boolean status;
    private List<PeopleData> peopleDataList;

    public String getLabel() {
        return label;
    }

    public void setLabel(String label) {
        this.label = label;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }

    public boolean isStatus() {
        return status;
    }

    public void setStatus(boolean status) {
        this.status = status;
    }

    public List<PeopleData> getPeopleDataList() {
        return peopleDataList;
    }

    public void setPeopleDataList(List<PeopleData> peopleDataList) {
        this.peopleDataList = peopleDataList;
    }

    public int getDepartmentId() {
        return departmentId;
    }

    public void setDepartmentId(int departmentId) {
        this.departmentId = departmentId;
    }

    @Override
    public int getId() {
        return departmentId;
    }

    @Override
    public String value() {
        return label;
    }
}

File: app/src/main/java/com/sdxxtop/zjlguardian/ui/department/data/DepartmentData.kt
package com.sdxxtop.zjlguardian.ui.department.data

import androidx.annotation.Keep

/**
 * Email: zhousaito@163.com
 * Created by zhousaito 2019-10-08 22:46
 * Version: 1.0
 * Description:
 */

@Keep
data class DepartmentData(
    val list: List<DepartmentDataItem>
)

@Keep
data class DepartmentDataItem(
    val add_date: String,
    val cat_id: Int,
    val cat_name: String,
    val duty_id: Int,
    val duty_name: String,
    val end_day: String,
    val event_id: Int,
    val status: String,
    val title: String
)